corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less

```diff
@@ -19,3 +19,89 @@
 .break-all-words {
     word-break: break-all;
 }
+// Bootstrap 3 does not have padding and margin classes, so adding these to utils 
+//https://stackoverflow.com/questions/32233489/does-bootstrap-have-builtin-padding-and-margin-classes
+
+.padding-xs { padding: .25em; }
```

Comment:
lots of raw, repeating numbers here. seems like a good candidate for less variables
cc @orangejenny